src/development/dev-branching-git-flow.md

```diff
@@ -4,6 +4,7 @@ title: "Git branching"
 description: "Define a branching strategy to avoid confusion and reduce risk"
 tags: [dev, branching]
 order: 8
```

Comment:
not related to this change directly but noticed the below sections may also need updating with the latest changes

- in coding-naming-conventions.md the Branch name regex

Reply:
Good point, we should have poicked that up in the previous PR.
That regex seems overly complex to me, but can we pick up in another PR please?

Reply:
Didn't get a chance to look at this yesterday but since you've already done the change i'll resolve it :)